### PR TITLE
burp-suite-free-edition 1.7.10 (new formula)

### PR DESCRIPTION
**Pull request by adam-moss**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

3rd question does not apply as this is a pre-built .jar file.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Is this open source software?

**Comment by adam-moss** [comment]
*facepalm* no, its not

File: Formula/burp-suite-free-edition.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  sha256 "9f609d14d474f43c9261c920c4a868278cbac119fcee62a328274e093738378d"
+
+  bottle :unneeded
+
+  depends_on :java => "1.6+"
+
+  def install
+    libexec.install "Download"
+
+    mv libexec/"Download", libexec/"burpsuite_free_v#{version}.jar"
```
Comment (bfontaine) [inline Formula/burp-suite-free-edition.rb]: The two lines above can be rewritten to `(libexec/"burpsuite_free_v#{version}.jar").install "Download"`.

**Comment by bfontaine** [comment]
>  **facepalm** no, its not 😭 

I’m afraid we can’t accept it in the core, then. You can still [maintain it in your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md). Sorry!

**Comment by adam-moss** [comment]
Yes, sorry about that.

Also thanks for taking the time to review and offer a code improvement, it's much appreciated!

**Pull request closed without merging**